spec/minimax_server/core_spec.clj

```diff
@@ -1,8 +1,8 @@
 (ns minimax-server.core-spec
-  (import [scarvill.httpserver.response ResponseBuilder])
-  (require [speclj.core :refer :all]
+  (:require [speclj.core :refer :all]
             [minimax-server.core :refer :all]))
 
-(describe "temporary tests"
-  (it "can use things from http server jar"
-    (should-be-a ResponseBuilder (new ResponseBuilder))))
+(describe "main"
```

Comment:
It looks like we are replacing temporary tests with different temporary tests. I would just delete this file until there are "real" tests that would live in here.